src/index.html

```diff
@@ -3,11 +3,286 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;700;800&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <h1 hidden>Bose</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__menu">
+          <a href="#" class="header__logo" >
+            <img
+              src="./images/logo.svg"
+              alt="BOSE logo"
+              class="header__logo-img"
+            >
+          </a>
+          <div class="header__navigation">
+            <a href="#" class="header__phone"></a>
+            <button class="header__menu-button header__burger"></button>
+            <button class="header__menu-button header__cross"></button>
+          </div>
+        </div>
+        <nav class="nav header__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link">Home</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">About Us</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Now on view</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Testimonials</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Contacts</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link nav__call">Call to order</a>
```

Comment:
should trigger a call href - both clicking the number or the link itself
![Snapshot 2022-04-12 12 13 38](https://user-images.githubusercontent.com/39308717/162925561-f3039f3a-333d-4def-95b5-4a94e783b6ce.png)